packages/fields/src/index.js

```diff
@@ -1,22 +1,23 @@
 export { Implementation } from './Implementation';
-
 export { default as CalendarDay } from './types/CalendarDay';
 export { default as Checkbox } from './types/Checkbox';
-export { default as CloudinaryImage } from './types/CloudinaryImage';
 export { default as Color } from './types/Color';
 export { default as DateTime } from './types/DateTime';
 export { default as Decimal } from './types/Decimal';
 export { default as File } from './types/File';
 export { default as Float } from './types/Float';
 export { default as Integer } from './types/Integer';
 export { default as Location } from './types/Location';
-export { default as OEmbed } from './types/OEmbed';
 export { default as Password } from './types/Password';
 export { default as Relationship } from './types/Relationship';
 export { default as Select } from './types/Select';
 export { default as Slug } from './types/Slug';
 export { default as Text } from './types/Text';
-export { default as Unsplash } from './types/Unsplash';
 export { default as Url } from './types/Url';
 export { default as Uuid } from './types/Uuid';
 export { default as Virtual } from './types/Virtual';
+
+// Moved out of core packages
+export { default as OEmbed } from '@keystonejs/fields-oembed';
+export { default as Unsplash } from '@keystonejs/fields-unsplash';
+export { default as CloudinaryImage } from '@keystonejs/fields-cloudinary-image';
```

Comment:
How can I show a deprecated warning when these are used in this way?

Reply:
what about a wrapper component which logs to console?

```js
import OEmbed as OEmbedComponent from '@keystonejs/fields-oembed';
export const OEmbed = props => {
   console.log('depricated');
    <OEmbed {...props} />; // we can add children as well
}
```

Reply:
That's not a component right? It's a Keystone field.

Reply:
🤦 
I think we may have to re-construct the field with view wrapper and extending the implementation with extra console log.

Reply:
Yeah I think you are right... I wanted avoid that if possible.